bureaucracy.js

```diff
@@ -27,6 +27,7 @@ function setup (fileinput, options) {
 function create (options) {
   var o = options || {};
   o.formData = o.formData || {};
+  o.fieldKey = o.fieldKey || {};
```

Comment:
oh haha duh, just a moment

On Mon, Aug 22, 2016 at 2:20 PM, Nicolás Bevacqua <notifications@github.com>
wrote:

> In bureaucracy.js
> <https://github.com/bevacqua/bureaucracy/pull/5#discussion_r75730382>:
>
> > @@ -27,6 +27,7 @@ function setup (fileinput, options) {
> >  function create (options) {
> >    var o = options || {};
> >    o.formData = o.formData || {};
> > +  o.fieldKey = o.fieldKey || {};
>
> Should default to 'uploads'
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bevacqua/bureaucracy/pull/5/files/8f8c08aafeeed62abfad42bc2781e3dac1b45fb9#r75730382>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABfJ4ORmZGyyoYMnPioJfMfKdlyxe-Tks5qiehngaJpZM4JqJxd>
> .
>